partner_create_by_vat/models/res_partner.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+# ©  2015 Forest and Biomass Services Romania
+# See README.rst file on addons root folder for license details
+
+import requests
+
+from string import maketrans
+from stdnum.eu.vat import check_vies
```

Comment:
Please import check_vies where you use it. That way, people who do not actually have the module installed do not have to have the stdnum module in their setup.